ui/application-window.ui

```diff
@@ -198,6 +198,17 @@ Author: Philipp Wolfer <ph.wolfer@gmail.com>
                 <property name="non_homogeneous">True</property>
               </packing>
             </child>
+            <child>
+              <object class="GtkLabel" id="time_label">
+                <property name="can_focus">False</property>
+                <property name="label" translatable="yes">label</property>
```

Comment:
You should mark this as non-translatable. Also please change the default "label" into "00:00", that way it looks nicer in glade.